services/blockchain-app-registry/tests/functional/shared/utils/downloadRepository.test.js

```diff
@@ -21,7 +21,7 @@ const {
 	getRepoDownloadURL,
 	getLatestCommitHash,
 	getCommitInfo,
-	getFileDownloadURL,
+	getFileDownloadUrl,
```

Comment:
Since `URL` is an abbreviation.
```suggestion
	getFileDownloadURL,
```

Reply:
Applied.